installer/src/orbic_network.rs

```diff
@@ -270,3 +270,16 @@ async fn setup_rayhunter(admin_ip: &str) -> Result<()> {
 
     Ok(())
 }
+
+/// Root the Orbic device and open an interactive shell
+pub async fn shell(
+    admin_ip: &str,
+    admin_username: &str,
+    admin_password: Option<&str>,
+) -> Result<()> {
+    start_telnet(admin_ip, admin_username, admin_password).await?;
+    eprintln!(
+        "This terminal is fairly limited. The shell prompt may not be visible, but it still accepts commands."
+    );
+    interactive_shell(admin_ip, 24, false).await
```

Comment:
this is a nitpick that probably extends outside the bounds of this PR's purview, but i'm noticing that these port numbers are hardcoded all over the installer code -- might be nice to extract them to a constant at some point